torch/ao/sparsity/sparsifier/base_sparsifier.py

```diff
@@ -142,16 +142,34 @@ def prepare(self, model, config):
                     else:
                         stack.append(child)
 
+        # TODO: Remove the configuration by reference ('module')
```

Comment:
would be good to do this soon, before users start depending on this

Reply:
I agree